index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+
+<html Lang="en-GB">
+
+<head>
+	
+	<link rel="stylesheet" type= "text/css" href="styles.css">
+	
+	<title>Ollie Ritchie, Portfolio</title>
+	
+</head>
+
+<body>
+	
+	<nav>
+		<div class="nav-bar">
+			<a href="#">About</a>
+			<a href="#">Portfolio</a>
+			<a href="#">Contact</a>
+		</div>
+		<button class=cv-download-button>Download CV</button>
+	</nav>
+	
+	<header class="welcome"> <!-- section contaning h1 title and a full width image -->
+		<h1 class=header-title>Ollie Ritchie</h1>
+		<img src="pexels-mads-thomsen-2739013-1.jpg" alt="image of a road in the wet green mountains">
+	</header>
+	
+	<main>	
+		<section class="portfolio"> <!-- section contating h2 heading and 4 protfolio cards -->
+			
+			<h2>Portfolio</h2>
+		
+			<div class="portfolio-card"> <!-- portfolio card No. 1 --> <!-- portfolio card made up of h3 heading, <p> description, img -->
+				<h3>Project title 1</h3>
+				<img src="green-leaves-1410259.jpg " alt="project 1 icon">
+				<p>
+					By an outlived insisted procured improved am. Paid hill fine ten now love even leaf. Supplied feelings mr of dissuade recurred no it offering honoured. Am of of in collecting devonshire favourable excellence.
+				</p>
+			</div>
+			
+			<div class="portfolio-card"> <!-- portfolio card No. 3 --> 
+				<h3>Project title 1</h3>
+				<img src="green-leaves-1410259.jpg " alt="project 1 icon">
+				<p>
+					By an outlived insisted procured improved am. Paid hill fine ten now love even leaf. Supplied feelings mr of dissuade recurred no it offering honoured. Am of of in collecting devonshire favourable excellence.
+				</p>
+			</div>
+			
+			<div class="portfolio-card"> <!-- portfolio card No. 2 --> 
+				<h3>Project title 1</h3>
+				<img src="green-leaves-1410259.jpg " alt="project 1 icon">
+				<p>
+					By an outlived insisted procured improved am. Paid hill fine ten now love even leaf. Supplied feelings mr of dissuade recurred no it offering honoured. Am of of in collecting devonshire favourable excellence.
+				</p>
+			</div>
+		
+		</section>
+	
+		<section class= "about">  <!-- section contating a breif bio and an image of me -->
+			<h2>About</h2>
+			<img src="IMG_4345.jpg" alt="image of ollie and his dog Monty">
+			<p>Yet bed any for travelling assistance indulgence unpleasing. Not thoughts all exercise blessing. Indulgence way everything joy alteration boisterous the attachment. Party we years to order allow asked of. We so opinion friends me message as delight. Whole front do of plate heard oh ought. His defective nor convinced residence own. Connection has put impossible own apartments boisterous. At jointure ladyship an insisted so humanity he. Friendly bachelor entrance to on by.
+			</p>
+		
+		</section>
+		
+	</main>
+	
+	<footer class="contact"> <!-- footer contaning h2 header, paragraph and links to socail inc icons -->
```

Comment:
As with header, you should only have one footer section, so perhaps no need to give it a class? Just select using ' footer'